README.md

```diff
@@ -3,59 +3,72 @@
 
-```go
-// Create client
-options := transloadit.DefaultConfig
-options.AuthKey = "TRANSLOADIT_KEY"
-options.AuthSecret = "TRANSLOADIT_SECRET"
-client, err := transloadit.NewClient(options)
-if err != nil {
-    panic(err)
-}
+This is a GO SDK to make it easy to talk to the [Transloadit](https://transloadit.com) REST API.
 
-// Initialize new assembly
-assembly := client.CreateAssembly()
-
-// Add input file to upload
-assembly.AddFile("myimage", "./lol_cat.jpg")
-
-// Add instructions, e.g. resize image to 75x75px
-assembly.AddStep("resize", map[string]interface{}{
-    "robot":           "/image/resize",
-    "width":           75,
-    "height":          75,
-    "resize_strategy": "pad",
-    "background":      "#000000",
-})
-
-// Wait until transloadit is done processing all uploads
-// and is ready to download the results
-assembly.Blocking = true
-
-// Start the upload
-info, err := assembly.Upload()
-if err != nil {
-    panic(err)
-}
+## Install
 
-fmt.Printf("You can view the result at: %s\n", info.Results["resize"][0].Url)
+```bash
+go get github.com/transloadit/go-sdk
 ```
 
-For examples on how to use templates, non-blocking processing and more, take a look at [`examples/`](https://github.com/transloadit/go-sdk/tree/master/examples).
+The Go SDK requires Go 1.1 or higher.
 
-## Installation
+## Usage
 
-```bash
-go get github.com/transloadit/go-sdk
+```go
+import (
+  "fmt"
+  "github.com/transloadit/go-sdk"
+)
+
+func main () {
+  // Create client
+  options := transloadit.DefaultConfig
+  options.AuthKey = "TRANSLOADIT_KEY"
+  options.AuthSecret = "TRANSLOADIT_SECRET"
+  client, err := transloadit.NewClient(options)
+  if err != nil {
+      panic(err)
+  }
+
+  // Initialize new assembly
+  assembly := client.CreateAssembly()
+
+  // Add input file to upload
+  assembly.AddFile("myimage", "/PATH/TO/FILE.jpg")
+
+  // Add instructions, e.g. resize image to 75x75px
+  assembly.AddStep("resize", map[string]interface{}{
+      "robot":           "/image/resize",
+      "width":           75,
+      "height":          75,
+      "resize_strategy": "pad",
+      "background":      "#000000",
+  })
+
+  // Wait until transloadit is done processing all uploads
+  // and is ready to download the results
+  assembly.Blocking = true
+
+  // Start the upload
+  info, err := assembly.Upload()
+  if err != nil {
+      panic(err)
+  }
+
+  fmt.Printf("You can view the result at: %s\n", info.Results["resize"][0].Url)
+}
 ```
 
-The Go SDK requires Go 1.1 or higher.
+## Example
+
+For fully working examples on how to use templates, non-blocking processing and more, take a look at [`examples/`](https://github.com/transloadit/go-sdk/tree/master/examples).
```

Comment:
@Acconut I made this a full path because based on the idea [here](https://github.com/transloadit/content/issues/228#issuecomment-292095624) this section might be pulled into the transloadit website, leaving the url broken on the website.